js/storage.js

```diff
@@ -0,0 +1,81 @@
+const userNameField = document.getElementById('name');
+const userEmailField = document.getElementById('email');
+const messageField = document.getElementById('message');
+
+const userInfo = {
+  name: localStorage.getItem('name'),
+  email: localStorage.getItem('email'),
+  message: localStorage.getItem('message'),
+};
+
+function checkUserNull() {
+  if (userInfo.name === null) {
+    userInfo.name = '';
+  }
+  if (userInfo.email === null) {
+    userInfo.email = '';
+  }
+  if (userInfo.message === null) {
+    userInfo.message = '';
+  }
+}
+
+function createLocalStorage() {
+  if (localStorage.getItem('name') === null && localStorage.getItem('email') === null && localStorage.getItem('message') === null) {
+    localStorage.setItem('name', '');
+    localStorage.setItem('email', '');
+    localStorage.setItem('message', '');
+  }
+}
+
+function loadLocalStorage() {
+  userNameField.value = localStorage.getItem('name');
+  userEmailField.value = localStorage.getItem('email');
+  messageField.value = localStorage.getItem('message');
+}
+
+function updateLocalStorage() {
+  localStorage.setItem('name', userInfo.name);
+  localStorage.setItem('email', userInfo.email);
+  localStorage.setItem('message', userInfo.message);
+}
```

Comment:
- [ ] Kindly here save all the data in one object as the requirements say `You must create a single JavaScript object with all the data from the entire form and save it in local storage. Do not create one local storage entry per input field.`.

Like 
```
const formData = {
    name: nameInput.value,
    email: emailInput.value,
    body: bodyInput.value,
};

```

And save this to the local storage `localStorage.setItem('formData ', formData);`